src/main/java/com/jungdam/AppController.java

```diff
@@ -1,12 +1,17 @@
 package com.jungdam;
 
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+@Api("Health")
 @RestController
 public class AppController {
+
+    @ApiOperation("Health Checking")
     @GetMapping("/health")
     public String getHealthCheck() {
```

Comment:
```suggestion
    public ResponseEntity<ResponseDto<HealthCheckDto>> getHealthCheck() {
```
나중에 이런식으로 refactoring을 해놓겠습니다. ㅎㅎ

Reply:
추후에 조금 더 간편하게 로직을 작성할게요 ㅜㅜ